docs/meeting-notes/foam-core-2020-07-11.md

```diff
@@ -0,0 +1,125 @@
+# Foam Core 2020-07-11
+
+Present: @jevakallio, @riccardoferretti
+
+### Tests
+
+- How do we know this approach works?
+  - Supports renaming
+  - Supports searching with (attribute-x)
+  - Find dead links
+
+### Getting started
+
+- Land work to master
+  - Create a foam-core package
+  -
+
+### Open questions
+
+- How should writing to files work
+  - What if affected notes have unsaved changes
+
+### Graph methods
+
+- get all
+- search by
+  - tag
+  - free text
+  - [[todo]]: how do vs code search editors work? are they pluggable? what do they need?
+- find dead links
+  - for linters
+- serialize/toJSON (for visualizers)
+- subscribe to changes
+
```

Comment:
find if a link exists (and which link) in a given row / column position + return it's start and end position - this would probably be needed e.g. to CTRL-hovering to work properly

Reply:
Can we add @jojanaho's commend to the list here, so we don't forget it?

```suggestion
  - find if a link exists (and which link) in a given row / column position + return it's start and end position - this would probably be needed e.g. to CTRL-hovering to work properly
```